plugins/source/mysql/resources/plugin/plugin_test.go

```diff
@@ -65,7 +67,7 @@ func createTable(ctx context.Context, db *sql.DB, table *schema.Table) error {
 		builder.WriteString(" PRIMARY KEY (")
 		for i, pk := range pks {
 			builder.WriteString(client.Identifier(pk))
-			if table.Columns.Get(pk).Type == schema.TypeString {
+			if table.Columns.Get(pk).Type == arrow.BinaryTypes.String {
```

Comment:
```suggestion
			if arrow.TypeEqual(table.Columns.Get(pk).Type, arrow.BinaryTypes.String) {
```
as otherwise you are comparing types and might get into trouble if someone uses `new(arrow.StringType)`